pkg/network/ovn_kubernetes.go

```diff
@@ -245,6 +245,8 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.Bo
 	data.Data["IPsecMachineConfigEnable"] = IPsecMachineConfigEnable
 	data.Data["OVNIPsecDaemonsetEnable"] = OVNIPsecDaemonsetEnable
 	data.Data["OVNIPsecEnable"] = OVNIPsecEnable
+	// Set RHEL_OS_CHECK to true only for non Hybrid hosted cluster.
+	data.Data["RHEL_OS_CHECK"] = bootstrapResult.Infra.HostedControlPlane == nil
```

Comment:
`RHEL_OS_CHECK` is not the most intuitive variable name.

Could we reuse something that we already have, like `{{- if not (eq .HostedClusterNamespace "")}}`?

Otherwise, if you think it adds value, I would be in favor of introducing a more explicit variable, like `HYPERSHIFT_HOSTED_CLUSTER"

Reply:
you are right, just reusing HostedClusterNamespace now.